root/layout/components/sidebar/WorkSidebar.js

```diff
@@ -77,6 +81,26 @@ const WorkSidebar = ({$c, work}: Props): React.Element<'div'> => {
               </SidebarProperty>
             ) : null}
 
+            {length == null ? null : (
+              <SidebarProperty
+                className="work-length"
+                label={addColonText(l('Median Length'))}
+              >
+                {formatTrackLength(length)}
+                {' '}
+                <InformationIcon
+                  className="align-top"
+                  title={l(
+                    `Autocalculated from all non-partial, non-medley,
+                     non-cover recordings linked only to this work.
+                     If the data seems off, it’s likely some
+                     recording relationships need to be marked as
+                     partial or medley.`,
```

Comment:
Or it is just the work can be played a random number of times, which is quite frequent, as already mentioned in comments to the ticket.

Reply:
Fair enough. Expanded the text a bit to indicate that possibility.